src/applications/mhv-secure-messaging/tests/e2e/pages/PatientComposePage.js

```diff
@@ -526,25 +526,61 @@ class PatientComposePage {
   };
 
   verifyRecipientsQuantityInGroup = (index, quantity) => {
-    cy.get(Locators.DROPDOWN.RECIPIENTS)
-      .find(`optgroup`)
-      .eq(index)
-      .find('option')
-      .should(`have.length`, quantity);
+    // Try combobox selector first (curated list flow), fallback to dropdown
+    cy.get('body').then($body => {
+      if ($body.find('[data-testid="compose-recipient-combobox"]').length > 0) {
```

Comment:
I think `cy.get('body')` may not be needed here. Also since you have `data-testid` in locator, you may just call `cy.findByTestId("compose-recipient-combobox")` instead. I think you may get away with the following: 
```suggestion
      if (cy.findByTestId("compose-recipient-combobox").length > 0) {
```